src/components/nodes/task/index.js

```diff
@@ -49,8 +102,28 @@ export default {
             },
           ],
         },
-        advancedAccordionConfig,
+        advancedAccordionConfigWithMarkerFlags,
       ],
     },
   ],
 };
+
+function getLoopCharacteristicsRadioValue(loopCharacteristics) {
+  if (!loopCharacteristics) {
+    return 'no_loop';
+  }
+
+  if (loopCharacteristics.$type === 'bpmn:StandardLoopCharacteristics') {
+    return 'loop';
+  }
+
+  if (loopCharacteristics.$type === 'bpmn:MultiInstanceLoopCharacteristics' && !loopCharacteristics.isSequential) {
+    return 'parallel_mi';
+  }
+
+  if (loopCharacteristics.$type === 'bpmn:MultiInstanceLoopCharacteristics' && loopCharacteristics.isSequential) {
+    return 'sequential_mi';
+  }
+
+  return 'no_loop';
+}
```

Comment:
This code is very similar to the code in the `inspectorHandler` method above. We could possibly pull it out into a function or some type of map object that associates `$type` to the inspector panel radio input values.

Reply:
Deferred as it's low return but fairly involved, so leaving it not DRY